src/backend.rs

```diff
@@ -1,18 +1,32 @@
+use std::fmt;
+use std::fmt::Formatter;
+
+pub use ::cpal::*;
```

Comment:
I haven't seen this syntax before. Would this also work?
```suggestion
pub use cpal::*;
```

Reply:
This is because Kira has a module also named `cpal` that was imported, the `::` prefix disambiguates between the crate (at the global namespace) vs. a submodule that is imported.

I think I might be able to simplify imports anyway and remove this syntax.

Reply:
Ah I see, thx